plugin/vcs/gitlab/gitlab.go

```diff
@@ -115,13 +124,19 @@ type FileCommit struct {
 	LastCommitID  string `json:"last_commit_id,omitempty"`
 }
 
+// RepositoryTreeNode is the API message for git tree node.
+type RepositoryTreeNode struct {
+	Path string `json:"path"`
+	Type string `json:"type"`
+}
+
 // FileMeta is the API message for file metadata.
 type FileMeta struct {
-	FileName     string `json:"file_name"`
-	FilePath     string `json:"file_path"`
-	Content      string `json:"content"`
-	Size         int64  `json:"size"`
-	LastCommitID string `json:"last_commit_id"`
+	FileName      string `json:"file_name"`
+	FilePath      string `json:"file_path"`
+	ContentBase64 string `json:"content"`
```

Comment:
Didn't catch this in the previous commit. FileMeta starts to diverge from the initial purpose.

Originally, it's just for metatdata for getting the LastCommitID. With the sheet integration, we probably need to rename it to File.

Reply:
And I suggest separating the method of getting file `data` vs getting file `metadata`.  The underlying implementation can be shared. This is similar to libc where we use `read` to read the content while using `stat` to read the metadata https://www.gnu.org/software/libc/manual/html_node/Reading-Attributes.html